watson/watson.py

```diff
@@ -425,3 +433,10 @@ def merge_report(self, frames_with_conflict):
                 merging.append(conflict_frame)
 
         return conflicting, merging
+
+    def _default_tags(self, project):
+        default_tags_raw = self.config.get('default_tags', project)
+        default_tags = []
+        if default_tags_raw is not None:
+            default_tags = shlex.split(default_tags_raw)
+        return default_tags
```

Comment:
I would factor this out into a new method of the `config.ConfigParser` class.

```
    def getlist(self, section, option, default=None):
        if not self.has_option(section, option):
            return [] if default is None else default
        
        value = self.get(section, option)
        
        if '\n' in value:
            return [item for item in value.splitlines() if item]
        else:
            return shlex.split(value)
```

This also allows an alternative syntax for lists of tags, some of which have spaces in them:

```
[default_tags]
interstellar travel = solar sail
    cryogenics
    fusion
watson =
    issue 113
    python
```